e2e/test/popver-tests/popover-test.js

```diff
@@ -0,0 +1,58 @@
+const assert = require('assert');
+const visualRegression = require('../../utils/visual-regression');
+const { takeScreenshotAndGetWholePageCompareResult, takeScreenShotOfElement } = visualRegression;
+
+//Selectors/xpaths
+const CLICK_ME_POPOVER_XPATH = '//*[@class="overlay-trigger"]//*[contains(text(),"Click me!")]';
+const OVERLAY_POPOVER_XPATH = '//*[@class="overlay-trigger overlay-trigger--active"]//*[@class="overlay popover"]';
+const TRIGERED_OVERLAY = '//*[@class="overlay-trigger overlay-trigger--active"]';
+const HOVER_ME_POPOVER_XPATH = '//*[@class="col col-3"][1]//*[@class="overlay-trigger"]//*[contains(text(),"Hover me!")]';
+const TRIGERED_OVERLAY_XPATH = '//*[@class="overlay-trigger overlay-trigger--active"]';
+
+//test names
+const CLICK_ME_POPOVER = 'click-popover';
+const OVERLAY_POPOVER = 'overlay-popover';
+const HOVER_ME_POPOVER = 'hover-popover';
+
+describe('FEF popover tests', () => {
+
+  it('Checks the the fef modal page title and browser compare visual regression', () => {
+    browser.url('/popover');
+    assert.equal(browser.getTitle(), 'Intellyo Application Design System');
+    assert(takeScreenshotAndGetWholePageCompareResult({defaultTolerance: 4, ignoreComparison: false}),
+      'Whole FEF popover page screenshot not similar to other configurations');
+  });
+
+  it('should check ' + CLICK_ME_POPOVER, () => {
+    browser.url('/popover');
+    assert.equal(browser.getTitle(), 'Intellyo Application Design System');
+    assert(browser.element(TRIGERED_OVERLAY).message
+    === 'An element could not be located on the page using the given search parameters.',
+    'Triggered overlay shows where its not allowed');
+    assert(takeScreenShotOfElement(CLICK_ME_POPOVER_XPATH,
+      {ffAndWindowsTolerance: 27, windowsTolerance: 19, defaultTolerance: 15.5, ignoreComparison: false}), //TODO: more than 20% tolerance
+      'Click me popover is not similar to reference picture');
+  });
+
+  it('should check ' + OVERLAY_POPOVER, () => {
+    browser.url('/popover');
+    assert.equal(browser.getTitle(), 'Intellyo Application Design System');
+    browser.click(CLICK_ME_POPOVER_XPATH);
+    assert(takeScreenShotOfElement(OVERLAY_POPOVER_XPATH,
+      {defaultTolerance: 11, ignoreComparison: false}),
+      'Overlay popover for click me is not similar to reference picture');
+  });
+
+  it('should check ' + HOVER_ME_POPOVER, () => {
+    browser.url('/popover');
+    assert.equal(browser.getTitle(), 'Intellyo Application Design System');
+    browser.moveToElement(HOVER_ME_POPOVER_XPATH);
+    browser.waitUntil(() =>
+      browser.element(TRIGERED_OVERLAY_XPATH).isExisting(),
+      15000,
+      'does not show call button');
+    assert(takeScreenshotAndGetWholePageCompareResult({defaultTolerance: 2, ignoreComparison: false}),
```

Comment:
dont you think that it would be wiser to have a reference image without the sidebar, so if that's changes (eg components get open) we wouldnt have a failing test?

Reply:
I have the following problem: I cannot take a picture without sidebar. There is no component for that, just for the "content" line on the up which not contains the whole "Lorem ipsum dolor sit amet, consectetur adipiscing elit." popover.

Reply:
Mh there's a selector for that: `//*[class="content"]`
<img width="1359" alt="screen shot 2018-01-23 at 21 18 59" src="https://user-images.githubusercontent.com/462848/35298662-2b532b7a-0083-11e8-9072-26ff3f2c2b8b.png">

Reply:
Here you can see this selector on the popovers page.
<img width="512" alt="image" src="https://user-images.githubusercontent.com/7290118/35299125-920fbf26-0084-11e8-902e-7e8228346b93.png">

Reply:
haha
![screenshot_20180123-213119](https://user-images.githubusercontent.com/462848/35299174-c7b81272-0084-11e8-964f-7eab4683740b.jpg)